### PR TITLE
addition of mobile feature to hawker page

### PR DESCRIPTION
**Pull request by yixin567**
I've edited the mobile features for the hawker pages and is currently waiting for the backend to add the images for each of the stall in the hawker center so for now the individual hawker page does not have the list of stalls yet. I will update it once it is done from the backend

### CODE REVIEW

File: src/pages/IndividualHawker/IndividualHawker.module.scss [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  padding: 0 12%;
+  padding-bottom: 50px;
+  background: #f9f8f4;
+}
+
+.hawker {
+  display: inline-grid;
+  grid-template-columns: auto auto auto;
+  grid-column-gap: 50px;
+  padding: 38px;
```
Comment (niqiukun) [inline src/pages/IndividualHawker/IndividualHawker.module.scss]: Can use a smaller padding on mobile. The current padding looks a bit too much when switched to mobile view.

image

File: src/pages/IndividualHawker/IndividualHawker.module.scss [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+  .hawker-address {
+    font-size: 12px;
+    line-height: 12px;
+    font-weight: 400;
+    margin-top: 6px;
+  }
+
+  .hawker-description {
+    margin-top: 10px;
+    font-size: 10px;
```
Comment (niqiukun) [inline src/pages/IndividualHawker/IndividualHawker.module.scss]: Can increase the font size a bit? Personally I feel that the minimal font size for the user to see the text clearly is 13px on mobile. You will get some spaces after reducing the 38px padding.

File: src/pages/IndividualHawker/IndividualHawker.module.scss [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+  }
+
+  .hawker-transport {
+    margin-top: 5px;
+    font-size: 8px;
```
Comment (niqiukun) [inline src/pages/IndividualHawker/IndividualHawker.module.scss]: Font size a bit too small?

**Review by niqiukun — APPROVED** [review]
LGTM except for some small padding / font issues.

**Comment by Kalsyc** [comment]
Looks good!

**Pull request merged**